actions/delete.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Default entity delete action
+ *
+ * @package Elgg
+ * @subpackage Core
+ */
+$guid = get_input('guid');
+$entity = get_entity($guid);
+
+if (!$entity instanceof ElggEntity) {
+	register_error(elgg_echo('entity:delete:item_not_found'));
```

Comment:
There is no entity, so we can't know the type and subtype to attempt translations.